src/metadata/MwdbIntegrate.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+import requests
+import urllib
+import mwdblib
+from mwdblib import Malwarecage
+from metadata import Metadata
+
+
+class MWDBIntegrateMetadata(Metadata):
+    __depends_on__ = ["CuckooBinariesMetadata"]
+
+    def __init__(self, mwdb_api_token: str) -> None:
+        super().__init__()
+        self.token = mwdb_api_token
+
+    def extract(self, matched_fname, dependent_meta):
+        if not dependent_meta.get("cuckoo_hash"):
+            return {}
+
+        hash = dependent_meta.get("cuckoo_hash")["value"]
+
+        cached = self.cache_fetch(hash)
+        if cached:
+            return cached
+
+        if not self.token:
+            logging.error(
+                "MWDB metadata not fetched, MWDB_API_TOKEN variable was not set"
+            )
+            return {}
+
+        try:
+            api = mwdblib.MalwarecageAPI(
+                api_url="https://mwdb.cert.pl/api/", api_key=self.token
```

Comment:
pass `api_url` from config (similarly to api_key)

Reply:
(in config, get api_url from MWDB_API_URL env variable, but use "https://mwdb.cert.pl/api/" as default value)